### PR TITLE
Show how to pass your own models in docs

### PR DESCRIPTION
**Pull request by Joshuafolorunsho**
This should close this [issue](https://github.com/mastra-ai/mastra/issues/893).
image
image

### CODE REVIEW

**Comment by calcsam** [comment]
We need to explain how people go about finding these packages. (They are community providers for the AI SDK). And presumably as well, what if there is not a community provider for the model they want (we want them to use Portkey, right?)

File: docs/src/pages/guide/how-to/00-llm-models.mdx [hunk 1]
```diff
@@ -79,7 +79,63 @@ You can choose from several providers and models:
...
+    {
+      role: "user",
+      content: "What is machine learning?",
+    },
+  ],
+  model: modelConfig,
+});
+```
+
+If you were using an agent. The process is the same.
```
Comment (calcsam) [inline docs/src/pages/guide/how-to/00-llm-models.mdx:136]: The fact that we need to say this is a sign that this text is in the wrong place. We need to create a Model Config reference file under a new LLM folder and move a lot of text there. Start with this bit for this PR but you will want to do a follow up moving the Model lists there as well.

Comment (Joshuafolorunsho) [reply docs/src/pages/guide/how-to/00-llm-models.mdx:136]: Completely agree. Working on the new llm reference, and porting all of these over.

Comment (Joshuafolorunsho) [reply docs/src/pages/guide/how-to/00-llm-models.mdx:136]: We don't need to merge this.

**Pull request merged**